build-system/pr-check.js

```diff
@@ -639,13 +639,12 @@ function main() {
       command.runVisualDiffTests();
       if (buildTargets.has('RUNTIME')) {
         command.buildRuntimeMinified(/* extensions */ false);
-        command.runBundleSizeCheck('pr');
-      } else {
-        command.runBundleSizeCheck('skipped');
       }
     } else {
-      // Generates a blank Percy build to satisfy the required Github check.
+      // Generates a blank Percy build and skip the bundle-size check to satisfy
+      // the required Github checks.
       command.runVisualDiffTests(/* opt_mode */ 'empty');
+      command.runBundleSizeCheck(/* action */ 'skipped');
```

Comment:
This has moved from the inner `if` block to the outer `if` block. Intentional?

On reading the code, I think it should stay in the inner block, since we want to skip the check only if `buildRuntimeMinified` was not called.

Reply:
This actually fixes a bug that we missed - before this change, `runBundleSizeCheck` would be executed if `buildTargets` has one of `INTEGRATION_TEST`, `RUNTIME`, `VISUAL_DIFF`, `FLAG_CONFIG`, `BUILD_SYSTEM` _and_ then based on whether `buildTargets` has/does not have `RUNTIME` it would run 'pr' or 'skipped' (respectively)

This means that if `buildTargets` doesn't have any of the above, `runBundleSizeCheck` wouldn't run at all, and the check would be stuck on queued.

After this change, `runBundleSizeCheck('skipped')` will run if non-of-the-above, and `runBundleSizeCheck('pr')` if `RUNTIME`

Reply:
Realized I had the if/elses all wrong. Fixed now. I think.